solr/core/src/test/org/apache/solr/response/TestPrometheusResponseWriter.java

```diff
@@ -78,7 +78,12 @@ public void testPrometheusStructureOutput() throws Exception {
           output.lines().filter(line -> !line.startsWith("#")).collect(Collectors.toList());
       filteredResponse.forEach(
           (actualMetric) -> {
-            String actualValue = actualMetric.substring(actualMetric.lastIndexOf("} ") + 1);
+            String actualValue;
+            if (actualMetric.contains("}")) {
+              actualValue = actualMetric.substring(actualMetric.lastIndexOf("} ") + 1);
+            } else {
+              actualValue = actualMetric.split(" ")[1];
+            }
```

Comment:
Also adding this in to better parse numeric value out of prometheus metrics based on whether or not tags exist